### PR TITLE
Fixing and integration testing the RDS encoder

### PR DESCRIPTION
**Pull request by patrickdbakke**

### CODE REVIEW

File: announcer.go
Comment (patrickdbakke) [inline announcer.go:41]: Change here represent an update in the `github.com/urfave/cli` library, and the "dps" "rds" typo.

File: announcer.go
Comment (patrickdbakke) [inline announcer.go:130]: I changed all the `log`s to `fmt` so they go to STDOUT and can get unit tested.

Comment (lund5000) [reply announcer.go:130]: I am not a Go expert, but a little research tells me that `log` is safe with concurrent go routines (which `grabAndSendData()` uses) where `fmt` is not. `log` can also be configured to write to STDOUT: `log.SetOutput(os.Stdout)`

That's more about learning, though. It seems like the risk of using `fmt` is where you have so many concurrent writes that the OS starts to interleave them, and I don't think we'll be anywhere near that limit here nor have a problem separating the messages if necessary.

File: announcer.go
Comment (lund5000) [inline announcer.go:233]: Nit: I don't think that `makeRDSMessage()` should be logging a message about _sending_ that message.

**Review by lund5000 — APPROVED** [review]
The changes to the code look good, though I don't approve of adding JavaScript tests to a Go project in the long run. I started working on replacing them with Go tests as a little learning project, but that's no reason to hold up trying to get the RDS messages sent in production.

**Comment by patrickdbakke** [comment]
> The changes to the code look good, though I don't approve of adding JavaScript tests to a Go project in the long run. I started working on replacing them with Go tests as a little learning project, but that's no reason to hold up trying to get the RDS messages sent in production.

Fair. Since they're built as integration tests, not unit tests, and build real connections to UDP and TCP servers, I'm less worried about it. 

This should free us up to use any language we want in the future. (ex: likely able to find more maintainers if we convert to JS or ruby).

File: announcer.go
Comment (patrickdbakke) [reply announcer.go:130]: Good call. I didn't dig to hard at go. This is an easy change to make.

**Pull request merged**
